pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -208,6 +223,32 @@ pub mod pallet {
 	pub type DomainHookAddress<T: Config> =
 		StorageMap<_, Blake2_128Concat, Domain, [u8; 20], OptionQuery>;
 
+	/// Storage for routers.
+	///
+	/// This can only be set by an admin.
+	#[pallet::storage]
+	#[pallet::getter(fn routers)]
+	pub type Routers<T: Config> = StorageMap<_, Blake2_128Concat, T::Hash, T::Router>;
```

Comment:
Does this storage overlap somehow with `DomainRouters`? Could/should the previous be removed?

Reply:
Yes, I'm also not sure if we should add this new storage or just change the old one and write a migration for it.

Reply:
Both solutions will need a migration anyways. But luckily, we can just purge the previous configuration and start using this new. So whatever you think it's simpler. 

I think any migration could be done after the audit, because the audit only requires to have the code submitted in the repo

Reply:
I also tend to be in favor of using the new one and dropping the old one. Mainly because this migration is easier than updating an existing storage to one with a different layout.